src/main/java/core/basesyntax/service/impl/FileReaderServiceImpl.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.FileReaderService;
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.List;
+
+public class FileReaderServiceImpl implements FileReaderService {
+    private static final String CAN_T_FIND_FILE_BY_PATH = "Can't find file by path ";
```

Comment:
Consider moving error messages to a separate constants class or an enum, as they might be reused across different classes, which would adhere to the DRY principle.